pkgs/development/compilers/rust/cargo.nix

```diff
@@ -7,12 +7,14 @@ rustPlatform.buildRustPackage rec {
   name = "cargo-${version}";
   inherit version;
 
-  src = fetchgit {
-    url = "https://github.com/rust-lang/cargo";
+  src = fetchFromGitHub {
```

Comment:
sorry mixed that up with rustc. I restored `fetchgit` for cargo even though the `rust-installer` submodule is not needed to compile cargo.